src/diffusers/models/transformer_2d.py

```diff
@@ -228,11 +230,15 @@ def forward(
             encoder_hidden_states ( `torch.LongTensor` of shape `(batch size, encoder_hidden_states dim)`, *optional*):
                 Conditional embeddings for cross attention layer. If not given, cross-attention defaults to
                 self-attention.
-            timestep ( `torch.long`, *optional*):
+            timestep ( `torch.LongTensor`, *optional*):
                 Optional timestep to be applied as an embedding in AdaLayerNorm's. Used to indicate denoising step.
             class_labels ( `torch.LongTensor` of shape `(batch size, num classes)`, *optional*):
                 Optional class labels to be applied as an embedding in AdaLayerZeroNorm. Used to indicate class labels
                 conditioning.
+            attention_mask ( `torch.Tensor` of shape (batch size, num latent pixels), *optional* ).
+                Bias to add to attention scores.
```

Comment:
I accepted this suggestion then reverted it, because I realised that `Transformer2DModel#forward` doesn't do any bool-to-bias handling. that stuff only happens in `UNet2DConditionModel#forward`, i.e. before we got here.

Reply:
Ah, ok. I got lost too.